schema/rdl_schema_0.1.json

```diff
@@ -4,180 +4,170 @@
 	"title": "Risk Data Library Schema",
 	"type": "object",
 	"required": [
-		"common"
+		"common",
+		"resources"
 	],
 	"properties": {
 		"common": {
```

Comment:
Is there a reason to keep the nesting under common?

If the schema represents a risk dataset, then the properties of `common` could all be top-level properties with the array of `resources` that make up the dataset as a sibling property.

Reply:
Discussion of nesting continued in https://github.com/GFDRR/rdl-standard/issues/57

Discussion of resources continued in https://github.com/GFDRR/rdl-standard/issues/54